src/contents/articles/hello-world/index.md

```diff
@@ -27,4 +25,4 @@ We want to grow as software developers but we also want to grow as a people. As
 
```

Comment:
**Why do we need a community?** is lacking a bit. I would not use my experience from Edinburgh, even though it has impacted me a lot. Don't want to encourage the colonial mentality in the nation and region. But more could be said about what we're lacking and the gap we're trying to fill